CONTRIBUTING.md

```diff
@@ -536,6 +536,12 @@ yarn test:mocha
 yarn test:karma:debug # remove the debug flag to run them once and close
 ```
 
+### Running Functional Tests 
+
```

Comment:
This is useful 👍 
What came to my mind when reading this, especially with this header, I'd expect info how to run functional tests (server & runner) in general, and then as follow up additional about default behavior (download) , and how to prevent the download.
What do you think?